sdk/nodejs/runtime/settings.ts

```diff
@@ -193,7 +209,7 @@ export function _setStack(val: string | undefined) {
  * monitor is a live connection to the resource monitor that tracks deployments (lazily initialized).
  */
 let monitor: any | undefined;
-const featureSupport: Record<string, boolean> = {};
+let featureSupport: Record<string, boolean> = {};
```

Comment:
Since we never re-assign `featureSupport` later on, I think it's correct to leave this as `const`

Reply:
Looks like it is assigned in `_reset()`